lib/src/modules/nft_v2.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:gql/language.dart';
+import 'package:graphql/client.dart';
+
+import 'package:fuse_wallet_sdk/fuse_wallet_sdk.dart';
+import 'package:fuse_wallet_sdk/src/utils/queries.dart';
+
+class NftModuleV2 {
+  final GraphQLClient _graphQLClient;
+
+  /// Creates an instance of [NftModule] with the given NFT subgraph URL.
+  ///
+  /// [nftSubgraph] is the NFT subgraph URL. Defaults to [https://api.thegraph.com/subgraphs/name/fuseio/fuse-nft-v2].
+  NftModuleV2({
+    String nftSubgraph = Variables.NFT_SUB_GRAPH_URL_V2,
+  }) : _graphQLClient = GraphQLClient(
+          link: HttpLink(nftSubgraph),
+          cache: GraphQLCache(),
+        );
+
+  /// Retrieves the collectibles owned by a specified address.
+  ///
+  /// [owner] is the owner's address.
+  ///
+  /// Returns a [Future] that resolves to a [DC] containing either an [OperationException] or a [Map] of collectibles.
+  Future<DC<OperationException?, Account>> getErc721(
```

Comment:
I'm have some hesitations:
- Do you think it's better to wrap the graph with our API backend and let access to it with only API keys? Since I see it's a single function used by the SDK it might not be difficult. 
- Do you think we still need the old module in the new SDK? There are still no active users except Voltage so it should be easy to make the necessary changes for them.
- If we need to have a v2 module, isn't it better to name the function the same as in the old class? 
Just thinking out loud, will be glad to hear your thoughts :)

Reply:
- We can do that. Actually, it makes much more sense.
- That makes sense, I'll remove it instead. 
- Good point.

I'll make the changes. And also expose the subgraph through the backend.

Reply:
Cool! 
If we're going for the backend we can take reference from Moralis NFT API: https://docs.moralis.io/web3-data-api/evm/nft-api